src/main/java/gndata/lib/util/Change.java

```diff
@@ -0,0 +1,25 @@
+package gndata.lib.util;
+
+import com.hp.hpl.jena.rdf.model.Model;
+
+/**
+ * Created by andrey on 30.04.15.
+ */
+public class Change {
+
+    Model change;
+    boolean positive;
```

Comment:
From the code below I assume, that "positive" denotes an addition to the model. The name "positive" does not really reflect this, maybe rename this to e.g. "isAddition" or something else that describes the function more specifically.